packages/flutter_tools/lib/src/commands/run_mojo.dart

```diff
@@ -19,9 +19,10 @@ const String _kDefaultBundlePath = 'build/app.flx';
 
 class RunMojoCommand extends FlutterCommand {
   final String name = 'run_mojo';
-  final String description = 'Run a Flutter app in mojo.';
+  final String description = 'Run a Flutter app in mojo (from github.com/domokit/mojo).';
+  final bool hideCommand;
```

Comment:
He'll need to change the ctor to something like:
```
RunMojoCommand({ bool hideCommand: false }) : _hideCommand = hideCommand {
...
}
```
which is a bit more verbose.

Could also do:
```
final bool hidden;

RunMojoCommand({ this.hidden: false }) {
...
}
```

which would override the getter from the parent Command class.